collector/index.md

```diff
@@ -0,0 +1,425 @@
+---
+title: 'pganalyze Collector'
+backlink_href: /docs
+backlink_title: 'Documentation'
+---
+
+The pganalyze collector periodically queries your database and sends metrics and metadata (as "snapshots") to
+the pganalyze app. This is the only component that talks directly to your database. For details on setting up
+the collector, see the [collector installation guide](/docs/install/02_installing_the_collector). The collector
+is open-source and the code is available [on GitHub](github.com/pganalyze/collector).
+
+
+# Configuration Settings
+
+The collector can be configured either through a `INI` config file (in a normal install) or environment variables
+(typically for running via its Docker image). Most settings can be configured through either mechanism (if both
+are present, the config file takes precedence). Note that a single collector instance can monitor more than one
+database server (though this is not supported when configuring through environment variables). The `[pganalyze]`
+section describes settings that apply to all servers, and the other sections are named after the database server
+they correspond to, and define settings for just that server. When using environment variables, the setting applies
+to all monitored servers.
+
+After you make changes, the config file can be reloaded while the collector is running with the command
+`pganalyze-collector --reload`. This minimizes monitoring interruptions and simplifies config file updates.
```

Comment:
"After you make changes, you can run `pganalyze-collector --test --reload` to verify the new configuration and reload the collector if they work correctly. This minimizes monitoring interruptions and simplifies config file updates."